lte/gateway/python/magma/pipelined/ebpf/ebpf_manager.py

```diff
@@ -306,22 +305,23 @@ def _get_if_name(self, if_index: int):
         return None
 
     def _get_mac_address(self, if_name: str):
-        addr_str = netifaces.ifaddresses(self.sgi_if_name)[netifaces.AF_LINK][0]['addr']
+        addr_str = get_mac_address(interface=if_name)
         LOG.debug("if-name: %s, mac: %s" % (if_name, addr_str))
         return self._pack_mac_addr(addr_str)
 
     def _get_mac_address_of_ip(self, ip_addr: IPAddress):
```

Comment:
These gw ips could be remote ips - Needs more investigation. These IPs come from mobilityD via GetGatewayInfo.

Reply:
After looking at this again - this was using netifaces before so this should not be dangerous.

Reply:
Are you sure? To me it seems like it was always scapy in this file.

Reply:
Conversation continued here:
https://github.com/magma/magma/pull/14355#discussion_r1022619162